packages/frontend-defaults/src/discovery.ts

```diff
@@ -81,7 +81,12 @@ export function getAvailableFeatures(config: Config): FrontendFeature[] {
   );
 }
 
-function isBackstageFeature(obj: unknown): obj is FrontendFeature {
+/**
+ * Checks whether a given value is a valid backstage frontend feature.
+ *
+ * @public
+ */
+export function isBackstageFeature(obj: unknown): obj is FrontendFeature {
```

Comment:
Hm if we are exporting this, I wonder if it belongs better in `@backstage/frontend-plugin-api`. @Rugvip?

Reply:
I'd be happy to move it there if it's better.